demisto_sdk/commands/content_graph/parsers/pack.py

```diff
@@ -126,9 +126,14 @@ def __init__(self, metadata: Dict[str, Any]) -> None:
         self.vendor_id: Optional[str] = metadata.get("vendorId")
         self.vendor_name: Optional[str] = metadata.get("vendorName")
         self.preview_only: Optional[bool] = metadata.get("previewOnly")
-        self.marketplaces: List[MarketplaceVersions] = (
+        self.marketplaces: List[str] = (
```

Comment:
@ilaner

Reply:
Consulted about this with @ilaner upon change.